content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -105,6 +106,26 @@ datasource db {
 
 Learn more about SQLite connection strings [here](/concepts/database-connectors/sqlite).
 
+#### Specify a CockroachDB data source
```

Comment:
I think we could keep PostgreSQL up and move cockroachdb down.
(We have a lot of users on PostgreSQL and cockroachdb is only preview)

Also the example would not work because it's missing the `previewFeatures` flag from the generator block

Reply:
Ah, yeah, there are two PostgreSQL examples and one had ended up at the bottom for some reason. I've moved them together and added the preview feature flag.

(This whole section is a bit muddled, it duplicates connection stuff we have elsewhere and we'd like to remove/single-source this eventually, but not fiddling with it yet...)